templates/flavors/azure-cni-v1/patches/azure-machine-template.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-md-0
+  namespace: default
+spec:
+  template:
+    spec:
+      networkInterfaces:
+      - subnetName: node-subnet-1
+        privateIPConfigs: 109 # 110 max IPConfigs per node
```

Comment:
why does the number (110) differ from the comment here (110)?

Reply:
Good catch, should be `110`. Updated.